Technomancy/app.ts

```diff
@@ -2689,6 +2690,10 @@ function load() {
             $('#building_' + type + " > .building_amount").html(format_num(buildings[type].amount, false));
         } 
     });
+    console.log("Loading scratchpad...");
+    if (localStorage.getItem("scratchpad")) {
+        $("#scratchpad")[0].value = localStorage["scratchpad"] || "";
```

Comment:
Is the || strictly necessary here? What's the issue with just using `localStorage["scratchpad"]`? Also, probably once again use `.val(string)` here.